### ISSUE TITLE
Vignette parameters_selection: Mixed and Bayesian models section clarification 

### ISSUE THREAD

**Issue by cmaimone** [easystats/parameters#276]
This vignette section https://easystats.github.io/parameters/articles/parameters_selection.html#mixed-and-bayesian-models is a bit confusing.  By running the code in the example with and without select_parameters(), it appears that when select_parameters() is used with lmer (at least with the example model), it simply has no effect.  This was not clear from the text in the section.  I'm guessing that "However, this procedures is not available for other types of models, such as mixed or Bayesian models." was meant to convey that, but I thought it was talking about a particular selection algorithm, not the entire function.

Is the effect the same for Bayesian models (not included in the vignette).  The documentation suggests it wouldn't be: "For Bayesian models, it uses the projpred package."

If select_parameters() is intended to simply have no effect for mixed (or other) models, separate from the vignette, it might also be nice to issue a warning to the user in such cases.

**Comment by strengejacke** [comment]
Thanks! We revised the vignette, which now explains more in detail how `select_parameters()` works for mixed models, slightly revised the examples, and added an example for Bayesian models to the vignette. It should now be more clear how `select_parameters()` works for mixed and Bayesian models.

Furthermore, we revised the docs of the `select_parameters()` function, so the help-file is more clear to users now.
